src/scenarios/shared/runner.py

```diff
@@ -514,6 +528,8 @@ def run(self):
                     getLogger().exception("Failed to get past permission screen, run locally to see if enough next button presses were used.")
                     exit(-1)
 
+            time.sleep(10) # Add delay to ensure app is fully installed and give it some time to settle
```

Comment:
Are you convinced that there's no way to directly observe the completion of whatever we're waiting for here? Totally fine if you are, but statements like this one read as "expect to change this number after eventual intermittent failures" to me.

Reply:
Yes, this is through trial an error, we've observed numbers are more stable if you leave an Android app on the screen for a bit before doing a startup test run.

I honestly don't see this number needing to change in the future. Nothing would fail if it is lower or higher -- we just might get more sporadic number if it is lower.